src/classes/controllers/CaseController.cls

```diff
@@ -0,0 +1,11 @@
+/**
+ * Created by Mariya on 29.01.2023.
+ */
+
+public with sharing class CaseController {
+
+    @AuraEnabled public static List<Case> getCasesByAccountId(Id accountId) {
```

Comment:
MAJOR
Why do you need a controller here? Controller layer is for UI/API communication.
Why is @AuraEnabled annotation added?